nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FTPTransfer.java

```diff
@@ -296,13 +296,19 @@ private FileInfo newFileInfo(final FTPFile file, String path) {
         perms.append(file.hasPermission(FTPFile.WORLD_ACCESS, FTPFile.READ_PERMISSION) ? "r" : "-");
         perms.append(file.hasPermission(FTPFile.WORLD_ACCESS, FTPFile.WRITE_PERMISSION) ? "w" : "-");
         perms.append(file.hasPermission(FTPFile.WORLD_ACCESS, FTPFile.EXECUTE_PERMISSION) ? "x" : "-");
+	
+	long lastModifiedTime		= System.currentTimeMillis();
```

Comment:
I'd be OK with seeing this as a one liner check/assignment:
`final long lastModifiedTime	= file.getTimestamp() != null ? file.getTimestamp().getTimeInMillis() : System.currentTimeMillis();`